### PR TITLE
do letsencrypt renew quietly

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by ekohl** [comment]
Why would you not do this weekly? It should check the expiration dates on the certificate and be a noop if nothing is about to expire.

**Comment by mmoll** [comment]
I'm not filtering any mails, so I'm now getting three mails each Saturday.

**Comment by ekohl** [comment]
How about using ```--quiet``` then? That's what Debian does by default as well. They even have a note that upstream recommends running twice a day.

**Comment by mmoll** [comment]
that sounds perfectly fine, changed 👍

File: puppet/modules/web/manifests/letsencrypt.pp [hunk 1]
```diff
@@ -7,7 +7,7 @@
   }
 
   cron { 'letsencrypt_renew':
-    command => "${::letsencrypt::command} renew --renew-hook '/sbin/service httpd reload'",
+    comman  => "${::letsencrypt::command} renew --quiet --renew-hook '/sbin/service httpd reload'",
```
Comment (ekohl) [inline puppet/modules/web/manifests/letsencrypt.pp]: You dropped a letter here.

Comment (mmoll) [reply puppet/modules/web/manifests/letsencrypt.pp]: fixed.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**